docs: update manually installing plugins docs (#13135)
hashicorp/packer: website/content/docs/plugins/install.mdx

## Manually install plugins using the CLI

You can use the `packer plugins install` command to manually install plugin binaries.

> Include the `--path` flag and specify a local source so that Packer automatically calculates the SHA256SUM file and installs the files into the Packer plugin directory:

Reviewer: ```suggestion
Use the `--path` flag to specify a local source. Packer will then automatically calculates the SHA256SUM file and install the files into the Packer plugin directory:
```
nit feel free to disregard but I just think this reads a bit cleaner

IMO this reads a bit cleaner

Author: Sounds good to me! I took some of the changes from your suggestion, feel free to take a look at the reroll and let me know if this looks good to you.
My editor's configured to remove trailing whitespaces automatically also, so feel free to disregard the second commit from the PR as it removes them all from that page.
